.github/workflows/bump-patch-version.yml

```
@@ -0,0 +1,79 @@
+      - name: Update version in TypeScript file
+        run: |
+          sed -i 's/version = "${{ env.current_version }}"/version = "${{ env.new_version }}"/' src/zl.ts
+
+      - name: Commit changes
+        run: |
+          git add package.json src/zl.ts
+          git commit -m "chore: bump version to ${{ env.new_version }}"
+          git push origin bump-version-${{ env.new_version }}
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.4)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)